plugins/chat/plugin.rb

```diff
@@ -295,10 +295,7 @@ module ::Chat
     end
 
     if name == :chat_allowed_groups
-      Jobs.enqueue(
-        Jobs::Chat::AutoRemoveMembershipHandleChatAllowedGroupsChange,
-        new_allowed_groups: new_value,
-      )
+      Jobs.enqueue(Jobs::Chat::AutoJoinUsers, event: "chat_allowed_groups_changed")
```

Comment:
We don't _need_ this to be a job, but in the odd chance the changes from the site setting triggers a huge change in memberships, I opted to leave it as is.